android/app/src/main/java/com/reactnativenavigation/views/sharedElementTransition/SharedElementTransition.java

```diff
@@ -80,7 +80,7 @@ private void saveTextViewSpannedText(final TextView view) {
             @Override
             public void run() {
                 if (spannableText == null) {
-                    spannedText = (SpannedString) view.getText();
+                    spannedText = new SpannedString(view.getText());
```

Comment:
finally was able to repro ... so, if i wrap TextInput in SharedElementTransition, i get 

java.lang.ClassCastException: android.text.SpannableStringBuilder cannot be cast to android.text.SpannedString at com.reactnativenavigation.views.sharedElementTransition.SharedElementTransition$2.run(SharedElementTransition.java:80)

since view is com.facebook.react.views.textinput.ReactEditText and view.getText() returns SpannableStringBuilder

This however doesnt happen if I wrap TextInput in transparent View